test/models/food_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class FoodTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
[REQUIRED] As suggested by the previous reviewer kindly remove all commented-out unused code.

Reply:
Thank you for the suggested changes @fobadara 
We will make the changes and submit for review.Thank you